RELEASE_NOTES.md

```diff
@@ -4,10 +4,21 @@ This page describes the noteworthy improvements provided by each release of Ecli
 
 ### Next release...
 
+## 0.10.1
+
+* 📅 Release Date (tentative): Early September 2020
+* All changes: https://github.com/eclipse/wildwebdeveloper/compare/0.10.0...0.10.1
+
+#### TypeScript prorgam debug support
+
+The TypeScript programs now can be directly debugged using `Node program` Debug Configuration.
```

Comment:
I would start the sentence with “TypeScript programs can now be directly debugged...” but this is subjective :)

Reply:
agreed